src/views/footnotes_element.ts

```diff
@@ -22,6 +22,16 @@ export class FootnotesElementView<
   PropsType extends BaseNodeProps
 > extends BlockView<PropsType> {
   public elementType = 'div'
+
+  onUpdateContent() {
```

Comment:
Because the methods of BlockView are defined is properties we can't access them in a successor using super. In a successor I need ideally to do the following:
`updateContents() {
      super.UpdateContents();
      // do some custom logic
}`

So as the things are right now I can't really do that. So I added that callback to be able just to customise the logic of updateContent without rewriting all the block elements in the app or without copying all the code into each successor.